frontend/composables/use-validators.ts

```diff
@@ -3,11 +3,18 @@ const EMAIL_REGEX =
 
 const URL_REGEX = /[-a-zA-Z0-9@:%._+~#=]{1,256}.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/;
 
+validate(what: string, value: string): boolean {
+	const response =  fetch(`/api/validators/${what}/${value}`);
+  const data = response.json();
+	return ! data.error;
+}
+
 export const validators: {[key: string]: (v: string) => boolean | string} = {
   required: (v: string) => !!v || "This Field is Required",
   email: (v: string) => !v || EMAIL_REGEX.test(v) || "Email Must Be Valid",
   whitespace: (v: string) => !v || v.split(" ").length <= 1 || "No Whitespace Allowed",
   url: (v: string) => !v || URL_REGEX.test(v) || "Must Be A Valid URL",
+	username: (v: string) => await
```

Comment:
I'm not sure that this will work with the current implementation of errors within Vuetify (the UI framework). I suggest we address this in another PR since it's probably going to require some custom error handling with advanced usage of the composition API. I know you're not super into TS/JS so I totally get it if you don't want to jump into this.

Reply:
sorry, that was a mistake. I did not want to commit that